articles/templates/articles/includes/article_teaser.html

```diff
@@ -1,7 +1,7 @@
 {% load static core_tags wagtailcore_tags wagtailimages_tags %}
 
 <article class="row teaser">
-    {% if article.main_image %}
+    {% if article.main_image and image != 'false' %}
```

Comment:
this is also included in topic_page.html and article_list_page.html, they should probably also pass in a value.